backend/src/main/java/com/carffeine/carffeine/station/domain/station/Coordinate.java

```diff
@@ -0,0 +1,48 @@
+package com.carffeine.carffeine.station.domain.station;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+public class Coordinate {
+
+    private final Latitude minLatitude;
+    private final Latitude maxLatitude;
+    private final Longitude minLongitude;
+    private final Longitude maxLongitude;
+
+    public static Coordinate from(BigDecimal latitude,
+                                  BigDecimal latitudeDelta,
+                                  BigDecimal longitude,
+                                  BigDecimal longitudeDelta) {
+        Latitude originLatitude = Latitude.from(latitude);
+        Latitude minLatitude = originLatitude.calculateMinLatitudeByDelta(latitudeDelta);
+        Latitude maxLatitude = originLatitude.calculateMaxLatitudeByDelta(latitudeDelta);
+
+        Longitude originLongitude = Longitude.from(longitude);
+        Longitude minLongitude = originLongitude.calculateMinLongitudeByDelta(longitudeDelta);
+        Longitude maxLongitude = originLongitude.calculateMaxLongitudeByDelta(longitudeDelta);
+
+        return new Coordinate(minLatitude, maxLatitude, minLongitude, maxLongitude);
+    }
+
+    public BigDecimal getMinLatitudeValue() {
+        return minLatitude.getValue();
+    }
+
+    public BigDecimal getMaxLatitudeValue() {
+        return maxLatitude.getValue();
+    }
+
+    public BigDecimal getMinLongitudeValue() {
+        return minLongitude.getValue();
+    }
+
+    public BigDecimal getMaxLongitudeValue() {
+        return maxLongitude.getValue();
+    }
```

Comment:
```suggestion
    public BigDecimal minLatitudeValue() {
        return minLatitude.getValue();
    }

    public BigDecimal maxLatitudeValue() {
        return maxLatitude.getValue();
    }

    public BigDecimal minLongitudeValue() {
        return minLongitude.getValue();
    }

    public BigDecimal maxLongitudeValue() {
        return maxLongitude.getValue();
    }
```
저희 게터 네이밍을 get을 빼고 이렇게 컨벤션을 정했던 것 같아요!